src/app/public/modules/viewkeeper/viewkeeper.ts

```diff
@@ -0,0 +1,378 @@
+import {
+  SkyViewkeeperOffset
+} from './viewkeeper-offset';
+
+import {
+  SkyViewkeeperOptions
+} from './viewkeeper-options';
+
+import {
+  SkyViewkeeperBoundaryInfo
+} from './viewkeeper-boundary-info';
+
+import {
+  SkyViewkeeperFixedStyles
+} from './viewkeeper-fixed-styles';
+
+const CLS_VIEWKEEPER_FIXED = 'sky-viewkeeper-fixed';
+const EVT_AFTER_VIEWKEEPER_SYNC = 'afterViewkeeperSync';
+
+let styleEl: HTMLStyleElement;
+let nextIdIndex: number;
+
+function ensureStyleEl(): void {
+  if (!styleEl) {
+    styleEl = document.createElement('style');
+
+    let css = document.createTextNode(`
+.${CLS_VIEWKEEPER_FIXED} {
+  position: fixed !important;
+  z-index: 999;
+  opacity: 0.95;
+  filter: alpha(opacity=95);
```

Comment:
I don't think we need the `filter` property anymore.
https://caniuse.com/#feat=css-opacity
```suggestion
```

Reply:
Fixed.